tests/network/expose.go

```diff
@@ -181,7 +170,8 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 	Context("Expose service on a VM", func() {
 		var tcpVM *v1.VirtualMachineInstance
 		BeforeEach(func() {
-			tcpVM = newLabeledVMI("vm", virtClient, true)
+			tcpVM = newLabeledVMI("vm")
+			tcpVM = tests.RunVMIAndExpectLaunch(tcpVM, 30)
```

Comment:
Originally, `WaitForSuccessfulVMIStartIgnoreWarnings` waits 180sec, but here you used 30sec.
Is it intentional?

Reply:
Well, I think that 30s is already way to long for a VM to launch. The typical boot time on my laptop is 5s.
If you believe 180s was a thoughtful number I don't mind using it.

Reply:
I think it depends on too many moving parts.
How about changing this in a separate PR which we can retry the tests there to see it is stable?

Reply:
I think it would be good to dedicate time to go over all timeouts. The reason is, we progressed so much that they are probably all wrong at this moment :) (E.g There was a time when storage was an issue. There was a time when we used bigger vms or "heavier" os)

Reply:
Fine, I've set this to 180s, and we'll have a discussion about this in the future.